docs/src/configuration.md

```diff
@@ -69,6 +69,37 @@ and the user-provided one are used in the user configuration if
 `stylix.base16Scheme` is not changed in the user config. If that is the case,
 only the user override is used.
 
+### Accessing
+
+When passing colors to unsupported targets, it is possible to access generated
+color scheme through `lib.stylix.colors`. It contains an attrset, populated by
+[base16.nix](https://github.com/SenchoPens/base16.nix)'s `mkSchemeAttrs`. TLDR
+on its contents is
+```nix
+lib.stylix.colors = {
+  base08 = "ff0000";
+  base08-hex-r = "ff";
+  base08-dec-r = "0.996094";
+  # ...
+  red = "ff0000";
+  # ...
+  withHashtag = {
+    base08 = "#ff0000";
+    # ...
+  };
+};
+```
+See `colors` function of `base16.nix` for more info.
+
+When using [`stylix.override`](#overriding), unmodified color scheme can be
+accessed from `lib.stylix.scheme` - which is otherwise identical to
+`lib.stylix.colors`.
```

Comment:
`config.lib.stylix.scheme` was added as a workaround to try and retain the generated palette file after running `nix-collect-garbage`, and AFAIK was never meant to be public.

Reply:
I see. This probably deserves a comment in the source code, since by itself the purpose of the attribute is confusing

Reply:
I believe that is what this comment a few lines below is referring to:

> generated.json is necessary in the presence of overrides.

Although it's not clear why `lib.stylix.scheme` is exported, when it could be defined as a local variable and only used to generate that file.

Reply:
Either way, this should be figured out separately from this PR